modules/datagrid/configuration/added/clustered-openshift.xml

```diff
@@ -144,11 +144,17 @@
                     <driver name="h2" module="com.h2database.h2">
                         <xa-datasource-class>org.h2.jdbcx.JdbcDataSource</xa-datasource-class>
                     </driver>
+                    <driver name="mysql" module="com.mysql">
```

Comment:
`mysql` and `postresql` drivers were intentionally removed for s390x as they are not available in RHEL8 images. Having there will probably lead to a startup issues

Reply:
removed `mysel` and `postresql` sections, plz review.